Makefile

```diff
@@ -1,4 +1,5 @@
 CONTROLLER_GEN ?= controller-gen
+CRD_VERSIONS ?= v1
```

Comment:
nit: most probably we won't need to redefine `CRD_VERSIONS` in command line. Better define it with `:=`.

The rest looks good to me!

Reply:
OK, done